diff -Nurb o.speak-1.13-source/Makefile.am n.speak-1.13-source/Makefile.am
--- o.speak-1.13-source/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ n.speak-1.13-source/Makefile.am	2006-09-03 17:14:22.000000000 +0200
@@ -0,0 +1,5 @@
+SUBDIRS = src
+pkgdatadir = $(datadir)/espeak-data
+install-data-local:
+	install -d $(DESTDIR)$(pkgdatadir)
+	cp -Rf espeak-data/* $(DESTDIR)$(pkgdatadir)
diff -Nurb o.speak-1.13-source/configure.ac n.speak-1.13-source/configure.ac
--- o.speak-1.13-source/configure.ac	1970-01-01 01:00:00.000000000 +0100
+++ n.speak-1.13-source/configure.ac	2006-09-03 17:51:34.000000000 +0200
@@ -0,0 +1,43 @@
+#                                               -*- Autoconf -*-
+# Process this file with autoconf to produce a configure script.
+
+AC_PREREQ(2.59)
+AC_INIT(speak, 1.13, )
+AM_INIT_AUTOMAKE
+AC_CONFIG_SRCDIR([src/speak.cpp])
+AC_CONFIG_HEADER([config.h])
+
+# Checks for programs.
+AC_PROG_CXX
+AC_PROG_CC
+
+# Checks for libraries.
+# FIXME: Replace `main' with a function in `-lportaudio':
+AC_CHECK_LIB([portaudio], [main])
+
+# Checks for header files.
+AC_HEADER_STDC
+AC_CHECK_HEADERS([locale.h stdlib.h string.h unistd.h wchar.h wctype.h portaudio.h])
+
+# Checks for typedefs, structures, and compiler characteristics.
+AC_C_CONST
+AC_C_RESTRICT
+AC_HEADER_STDBOOL
+
+# Checks for library functions.
+AC_FUNC_CLOSEDIR_VOID
+AC_FUNC_MALLOC
+AC_FUNC_MEMCMP
+AC_FUNC_REALLOC
+AC_TYPE_SIGNAL
+AC_FUNC_STAT
+AC_CHECK_FUNCS([memset pow setlocale sqrt strchr Pa_IsStreamActive])
+
+if test "$datadir" = "\${prefix}/share" ; then
+	datadirectory=$ac_default_prefix/share
+else
+	datadirectory="$datadir"
+fi
+AC_DEFINE_UNQUOTED(DATADIRECTORY, "$datadirectory", [data directory])
+AC_CONFIG_FILES([Makefile src/Makefile])
+AC_OUTPUT
diff -Nurb o.speak-1.13-source/src/Makefile.am n.speak-1.13-source/src/Makefile.am
--- o.speak-1.13-source/src/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ n.speak-1.13-source/src/Makefile.am	2006-09-02 21:16:02.000000000 +0200
@@ -0,0 +1,17 @@
+AM_CFLAGS = -Wall -pedantic -fno-exceptions
+bin_PROGRAMS = speak
+speak_SOURCES =  \
+	compiledict.cpp \
+	dictionary.cpp \
+	intonation.cpp \
+	readclause.cpp \
+	setlengths.cpp \
+	speak.cpp \
+	synthdata.cpp \
+	synthesize.cpp \
+	tr_english.cpp \
+	tr_languages.cpp \
+	translate.cpp \
+	voices.cpp \
+	wavegen.cpp
+
diff -Nurb o.speak-1.13-source/src/speak.cpp n.speak-1.13-source/src/speak.cpp
--- o.speak-1.13-source/src/speak.cpp	2006-08-30 04:34:17.000000000 +0200
+++ n.speak-1.13-source/src/speak.cpp	2006-09-03 17:31:31.000000000 +0200
@@ -27,7 +27,7 @@
 #include <time.h>
 #include <signal.h>
 #include <locale.h>
-#include "sys/stat.h"
+#include <sys/stat.h>
 
 #include "speech.h"
 #include "voice.h"
@@ -36,6 +36,7 @@
 #include "translate.h"
 #include "speak_lib.h"
 
+#include "config.h"
 
 
 char path_home[120];
@@ -158,7 +159,7 @@
 	sprintf(path_home,"%s/espeak-data",getenv("HOME"));
 	if(access(path_home,R_OK) != 0)
 	{
-		strcpy(path_home,"/usr/share/espeak-data");
+		strcpy(path_home, DATADIRECTORY "/espeak-data");
 	}
 #endif
 }
diff -Nurb o.speak-1.13-source/src/voices.cpp n.speak-1.13-source/src/voices.cpp
--- o.speak-1.13-source/src/voices.cpp	2006-08-30 04:34:17.000000000 +0200
+++ n.speak-1.13-source/src/voices.cpp	2006-09-02 19:13:45.000000000 +0200
@@ -18,15 +18,15 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
-#include "stdio.h"
-#include "ctype.h"
-#include "wctype.h"
-#include "string.h"
-#include "stdlib.h"
-#include "speech.h"
+#include <stdio.h>
+#include <ctype.h>
+#include <wctype.h>
+#include <string.h>
+#include <stdlib.h>
+#include <speech.h>
 
 #ifndef PLATFORM_RISCOS
-#include "dirent.h"
+#include <dirent.h>
 #endif
 
 #include "voice.h"
diff -Nurb o.speak-1.13-source/src/wavegen.cpp n.speak-1.13-source/src/wavegen.cpp
--- o.speak-1.13-source/src/wavegen.cpp	2006-08-30 04:34:17.000000000 +0200
+++ n.speak-1.13-source/src/wavegen.cpp	2006-09-03 18:05:06.000000000 +0200
@@ -33,13 +33,16 @@
 #include "synthesize.h"
 
 #ifdef USE_PORTAUDIO
-#include "portaudio.h"
+#warning USE_PORTAUDIO IS DEFINED
+#include <portaudio.h>
 #undef USE_PORTAUDIO
 // determine portaudio version by looking for a #define which is not in V18
 #ifdef paNeverDropInput
 #define USE_PORTAUDIO   19
+#warning USE_PORTAUDIO IS DEFINED AS 19
 #else
 #define USE_PORTAUDIO   18
+#warning USE_PORTAUDIO IS DEFINED AS 18
 #endif
 #endif
 
